tests/contrib/contrib.py

```diff
@@ -0,0 +1,57 @@
+"""
```

Comment:
We can brainstorm here for any other testcases that would be worth adding that each integration should extend/implement.